### ISSUE TITLE
tables formating

### ISSUE THREAD

**Issue by jarekgajewski** [vantagedesign/ace-documentation#2]
Tables facing problems with header formatting. Frames are missing. 
All below header row is formatted right

**Comment by julianflapper** [comment]
@jarekgajewski can you give a code or image example of what's missing?

**Comment by jarekgajewski** [comment]
@julianflapper 
here is it:

Code:
|    Term    | Meaning           |
| :--------: | ----------------- |
|    MUST    | The definition is an absolute requirement of the specification.   |
screen
![image](https://user-images.githubusercontent.com/48130892/80725484-6f7f5500-8b03-11ea-92e6-79ed7f3e3ea0.png)

**Comment by julianflapper** [comment]
I see.. strange.
I'll look into it, thanks for reporting this!

**Comment by julianflapper** [comment]
This will be fixed in v1.0.3 (already on dev branch), using a table shortcode:
```
{{< table >}}
|    Term    | Meaning           |
| :--------: | ----------------- |
|    MUST    | The definition is an absolute requirement of the specification.   |
{{< /table >}}
```
this wraps the table content in a 'table' class to provide the styling.
